src/components/ButtonPanel.js

```diff
@@ -1,42 +1,51 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
 
-const ButtonPanel = () => (
-  <div className="flex flex-col btnPanel">
-    <div className="rowPanel">
-      <Button color="#d3d3d3" buttonName="AC" />
-      <Button color="#d3d3d3" buttonName="+/-" />
-      <Button color="#d3d3d3" buttonName="%" />
-      <Button buttonName="÷" />
-    </div>
+const ButtonPanel = props => {
```

Comment:
- [ ] [OPTIONAL] You can destruct the props at this line to avoid repeating `props.` later:
`const ButtonPanel = { clickHandler } => {`
`const handleClickBP = buttonName => clickHandler(buttonName); `